provider/cldr/src/transform/dates/patterns.rs

```diff
@@ -159,18 +161,40 @@ impl From<&cldr_json::DateTimeFormats> for gregory::patterns::DateTimeFormatsV1
                         },
                     };
 
-                    if !variant_parts.is_empty() {
-                        eprintln!(
-                            "This skeleton string is not yet supported: {:?}",
-                            skeleton_str
-                        );
-                        continue;
-                    }
-
-                    let pattern = pattern::Pattern::from_bytes(pattern_str as &str)
+                    let pattern = Pattern::from_bytes(pattern_str as &str)
                         .expect("Unable to parse a pattern");
 
-                    skeletons.0.insert(skeleton_fields_v1, pattern.into());
+                    match variant_parts.as_slice() {
```

Comment:
issue: can we have some comments explaining the format being parsed here

Reply:
Done